SDL/MiniDoublet.cu

```diff
@@ -8,10 +8,77 @@ void SDL::miniDoublets::resetMemory(unsigned int nMemoryLocationsx, unsigned int
 {
     cudaMemsetAsync(anchorHitIndices,0, nMemoryLocationsx * 3 * sizeof(unsigned int),stream);
     cudaMemsetAsync(dphichanges,0, nMemoryLocationsx * 9 * sizeof(float),stream);
-    cudaMemsetAsync(nMDs,0, (nLowerModules + 1) * sizeof(unsigned int),stream);
+    cudaMemsetAsync(nMDs,0, (nLowerModules + 1) * sizeof(int),stream);
     cudaMemsetAsync(totOccupancyMDs,0, (nLowerModules + 1) * sizeof(unsigned int),stream);
 }
 
+    //#ifdef CUT_VALUE_DEBUG
+//ALPAKA_FN_ACC void addMDToMemory(struct miniDoublets& mdsInGPU, struct hits& hitsInGPU, struct modules& modulesInGPU, unsigned int lowerHitIdx, unsigned int upperHitIdx, uint16_t& lowerModuleIdx, float dz, float drt, float dPhi, float dPhiChange, float shiftedX, float shiftedY, float shiftedZ, float noShiftedDz, float noShiftedDphi, float noShiftedDPhiChange, float dzCut, float drtCut, float miniCut, unsigned int idx)
+//#else
```

Comment:
In my opinion, it seems ok that we leave this comment here, since it's for optimization use. If we want to optimize the MD requirement at some point, this flag would be useful that we can enable it and get some output branches. But on the other hand, since it's alpaka branch instead of master branch, we probably won't do optimizations on this branch, so it maybe also good if we remove them :)

Reply:
Thanks, Yanxi! On your last point, imagine that at some point the alpaka variant will replace the master and this would mean that this gets lost and then perhaps difficult to retrace. So, if it's needed, let's not delete it.

However, what I am doubting here is the necessity of code that's commented out. If we intend to use it, then this should be stripped down to only the vital portions of the relevant code and be commented in - after all, it is run with a separate compilation flag.

Reply:
The comments were included on the master branch, so I included them in the commit. If they should be removed, this is something that should be done on the master branch first, I think.

Reply:
Sure, let's not address them here.